72pe.typ

```diff
@@ -168,9 +261,130 @@
 // -----------------------------------------------------------------------------
 // Ground Checklists and Information page
 // -----------------------------------------------------------------------------
-#let ground_checklists_and_info = [
+#let ground_checklists_and_info_columns = [
+	#checklist("Preflight", palette.brown,
+		checklist_group("Fluids"),
+		([Engine master], [OFF]),
+		([Fuel quantity], [CHECK VISUALLY]),
+		([Fuel sumps], [DRAIN],
+		 [Left wing, right wing, fuel strainer. Inspect for contamination.]),
+		([Fuel filler caps], [SECURE]),
+		([Engine oil level], [CHECK], [Minimum 6 quarts]),
+		([Gearbox oil level], [CHECK], [Oil must cover at least 1/2 of inspection glass]),
+		checklist_group("Cabin"),
+		([Pitot cover], [REMOVE]),
+		([POH], [VERIFY PRESENT]),
+		([Master], [ON]),
+		([Flaps], [EXTEND]),
+		([Fuel gauges], [CHECK QUANTITY]),
+		([Water level light], [CHECK OFF]),
+		([Avionics], [ON], [Check avionics fan audible]),
+		([Avionics], [OFF]),
+		([Annunciator panel], [TST]),
+		([Lights], [CHECK]),
+		([Pitot heat], [TEST]),
+		([Master], [OFF]),
+		([Fire extinguisher], [SECURE]),
+		([Control lock], [REMOVE]),
+		checklist_group("Empennage"),
+		([Rudder gust lock], [REMOVE]),
+		([Control surfaces], [CHECK]),
+		checklist_group("Right"),
+		([Main wheel tire], [CHECK INFLATION]),
+		([Aileron], [CHECK]),
+		checklist_group("Nose"),
+		([Propeller, spinner], [CHECK]),
+		([Air filter], [CHECK CLEAR]),
```

Comment:
2PE's POH just has air inlet obstruction checks, not specifically an air filter check.